VideoLocker/src/main/java/org/edx/mobile/module/db/DatabaseModelFactory.java

```diff
@@ -70,6 +73,7 @@ public static IVideoModel getModel(VideoResponseModel vrm) {
         e.videoId = vrm.getSummary().getId();
         e.transcript = vrm.getSummary().getTranscripts();
         e.lmsUrl = vrm.unit_url;
+        e.isVideoForWebOnly =  new Random().nextBoolean();//vrm.webOnly == 1;
```

Comment:
@BenjiLee I should add a comment here. it is for testing only. I need to remove it.